tests/utils/test_downstream.py

```diff
@@ -0,0 +1,134 @@
+"""Tests for tasks related to getting downstream artifacts"""
+import tamr_toolbox
+from tests._common import get_toolbox_root_dir
+from tamr_toolbox.utils.testing import mock_api
+
+CONFIG = tamr_toolbox.utils.config.from_yaml(
+    f"{get_toolbox_root_dir()}/tests/mocking/resources/toolbox_test.yaml"
+)
+MASTERING_UNIFIED_DATASET_ID = CONFIG["datasets"]["minimal_mastering_unified_dataset"]
+SOURCE_DATASET_ID = CONFIG["datasets"]["people_tiny.csv"]
+ONLINE_TEST = False
+
+
+@mock_api(enforce_online_test=ONLINE_TEST)
```

Comment:
enforce_online should only be used for local testing. Please update to @mock_api() and remove the global ONLINE_TEST